fe/fe-core/src/main/java/org/apache/doris/alter/CloudRollupJobV2.java

```diff
@@ -124,6 +124,15 @@ protected void onCancel() {
             try {
                 ((CloudInternalCatalog) Env.getCurrentInternalCatalog())
                     .dropMaterializedIndex(tableId, rollupIndexList, false);
+                for (Map.Entry<Long, Map<Long, Long>> partitionEntry : partitionIdToBaseRollupTabletIdMap.entrySet()) {
+                    Long partitionId = partitionEntry.getKey();
+                    Map<Long, Long> rollupTabletIdToBaseTabletId = partitionEntry.getValue();
+                    for (Map.Entry<Long, Long> tabletEntry : rollupTabletIdToBaseTabletId.entrySet()) {
+                        Long baseTabletId = tabletEntry.getValue();
+                        ((CloudInternalCatalog) Env.getCurrentInternalCatalog())
+                            .removeSchemaChangeJob(dbId, tableId, baseIndexId, partitionId, baseTabletId);
+                    }
+                }
```

Comment:
log here:
* which table/index has been processed (id name etc.)
* how many talbets have been processed here.

Reply:
Ms will check the message <origin_idx, rollup_idx, origin_tablet, rollup_tablet>. If it's not match, skip the request.